core/app/models/spree/promotion_handler/free_shipping.rb

```diff
@@ -19,13 +22,12 @@ def activate
 
       private
 
-        def promotions
-          Spree::Promotion.active.where({
-            :id => Spree::Promotion::Actions::FreeShipping.pluck(:promotion_id),
-            :code => nil,
-            :path => nil
-          })
-        end
+      def promotions
+        Spree::Promotion.active.where({
+          id: Spree::Promotion::Actions::FreeShipping.pluck(:promotion_id),
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.